File: 02-spring-boot-data/jdbc-template/src/main/java/vip/yazilim/basics/jdbc/service/IPersonService.java
package vip.yazilim.basics.jdbc.service;

import java.util.List;

import vip.yazilim.basics.jdbc.entity.Person;

/**
 * Person Repository to get data from database
 * 
 * @author Emre Sen - Aug 18, 2018
 *
 */
public interface IPersonService {

	// ...
	int insert(Person person);

	boolean update(Person person);

	int delete(int id);

	// ...
	List<Person> findAll();

	Person findById(int id);

}

File: spring-boot-thymeleaf/04-thymeleaf-form-submission/src/main/kotlin/vip/yazilim/thymeleaf/form_submission/service/IPersonService.kt
package vip.yazilim.thymeleaf.form_submission.service

import vip.yazilim.thymeleaf.form_submission.model.Person

/**
 *
 * @author maemresen - maemresen@yazilim.vip
 * 27.12.2020
 */
interface IPersonService {
    fun getAll(): List<Person>
    fun getByName(name: String): Person
}